src/sourmash/minhash.py

```diff
@@ -682,6 +682,22 @@ def to_frozen(self):
         new_mh.__class__ = FrozenMinHash
         return new_mh
 
+    def inflate(self, from_mh):  
+        if (self.track_abundance == True) and (from_mh.track_abundance == True):
```

Comment:
note, this can be shortened to `if self.track_abundance and from_mh.track_abundance`